packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-number/gux-form-field-number.tsx

```diff
@@ -158,6 +164,10 @@ export class GuxFormFieldNumber {
     this.hasError = hasSlot(this.root, 'error');
     this.hasHelp = hasSlot(this.root, 'help');
 
+    if (this.hasError) {
+      this.input.setAttribute('aria-invalid', 'true');
+    }
+
```

Comment:
Would a @Watch on `hasError` work instead of repeating code like this in different functions?